packages/angular/.storybook/main.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2016-2020 VMware, Inc.
+ * All Rights ReserveThis software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+const TsConfigPathsPlugin = require('tsconfig-paths-webpack-plugin');
+
+module.exports = {
+  stories: ['../src/**/*.stories.ts'],
+  addons: [
+    '@storybook/addon-a11y',
```

Comment:
I think we should disable this for the time being.

Reply:
Removed for now.